test/lib/glob.js

```diff
@@ -189,3 +188,209 @@ test("glob subdirectory", (t) => {
 		});
 	});
 });
+
+test("glob with multiple patterns with static exclude", (t) => {
+	t.plan(4);
+	return new FsAdapter({
+		fsBasePath: "./test/fixtures/glob",
+		virBasePath: "/test-resources/",
+		excludes: [
+			"/test-resources/application.b/**",
+			"!/test-resources/application.b/**/manifest.json"
+		]
+	}).byGlob(["/**/*.yaml", "/test-resources/**/i18n_de.properties"])
+		.then((resources) => {
+			const expectedResources = [
+				"/test-resources/application.a/ui5.yaml",
+				"/test-resources/application.b/webapp/manifest.json",
+				"/test-resources/application.b/webapp/embedded/manifest.json"
+			];
+			matchGlobResult(t, resources, expectedResources);
+		});
+});
+
+/* Generic Micromatch tests for understanding GLOB exclude behavior*/
```

Comment:
To be removed or do we want to enable them?

Reply:
Clarified in the comment. I would like to keep them as they might become helpful in future glob madness work